libs/pages/elewa/about-us/src/lib/team-members/team-members.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="team-wrapper">
+  <div class="header">
+    <h2>Management</h2>
+    <h2>Team</h2>
+  </div>
+  <div class="team-main">
+    <div class="team-cards" *ngFor="let member of membersArr">
+      <div class="card">
+        <img
+          src="{{ member.imageLink }}"
```

Comment:
use [src]="member.imageLink" instead of src="{{ member.imageLink }}"

do the same for all image src tags here.